lib/web_ui/dev/test_platform.dart

```diff
@@ -540,18 +540,18 @@ class BrowserPlatform extends PlatformPlugin {
 
       final String testRunner = isWasm ? '/test_dart2wasm.js' : 'packages/test/dart.js';
 
-      final String canvasKitVariant = getCanvasKitVariant();
       return shelf.Response.ok('''
         <!DOCTYPE html>
         <html>
         <head>
           <title>${htmlEscape.convert(test)} Test</title>
           <meta name="assetBase" content="/">
           <script>
-            window._flutter_canvaskit_variant_for_test_only = "$canvasKitVariant";
```

Comment:
I am just trying to avoid adding more special global stuff to the configuration of the flutter app. In this case, it feels extra redundant because we're already passing it through the engine configuration call a few lines below.